apps/ledger-live-mobile/src/components/SelectDevice2/index.tsx

```diff
@@ -39,9 +39,12 @@ type Navigation = BaseComposite<
 
 type Props = {
   onSelect: (_: Device) => void;
+  // Needed to know if a device has been selected and with which we are communicating with.
+  // The scanning can only occur when we are not communicating with a device.
+  isDeviceSelected: boolean;
```

Comment:
I personally would change the name of this prop so the users of the component can properly understand the impact it has. maybe "stopBleScanning" too, just like on the hook. wdyt?